### ISSUE TITLE
2.1 resume training gives ERRORS

### ISSUE THREAD

**Issue by LIQUIDMIND111** [TheLastBen/fast-stable-diffusion#2369]
1.5 training gives NO errors when resuming.

2.1 has given this errors many times before and now again:

its a conversion error
![Screenshot 2023-07-22 074352](https://github.com/TheLastBen/fast-stable-diffusion/assets/112510084/60dfdb77-b87d-4487-a37e-89bd01fb3d4a)

**Comment by TheLastBen** [comment]
that's an issue with disk storage, make sure everything is fine in that department

**Comment by LIQUIDMIND111** [comment]
> that's an issue with disk storage, make sure everything is fine in that department

i have 7gb available for storage. the ckpt files are less than 4GB, how much space i need so i can resume training?

**Comment by LIQUIDMIND111** [comment]
> that's an issue with disk storage, make sure everything is fine in that department

i just removed my older files on DRIVE, and same errors.. it mentions a conversion error. what does that has to do with storage? i have plenty of storage.

**Comment by TheLastBen** [comment]
the model is corrupt

**Comment by LIQUIDMIND111** [comment]
> the model is corrupt

it has been doing this for months.... maybe the code has errors, and since not many people use 2.1. not many complain.... file cannot be corrupt..... its works on my sd but needs more training, so..... maybe its a code issue...

**Comment by LIQUIDMIND111** [comment]
if its being corrupted, its a training issue then..... 1.5 works well when training.....

**Comment by TheLastBen** [comment]
are you sure you have enough space in your gdrive ?

**Comment by LIQUIDMIND111** [comment]
> are you sure you have enough space in your gdrive ? and are you using the latest notebook ?

yes, i have 10gb now. i have trained 2.1 before and ONLY get this when resuming.....

**Comment by LIQUIDMIND111** [comment]
> are you sure you have enough space in your gdrive ? and are you using the latest notebook ?

https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Comment by TheLastBen** [comment]
I tried training v2.1, reloaded the session, the model converted fine

**Comment by LIQUIDMIND111** [comment]
> I tried training v2.1, reloaded the session, the model converted fine

and how much FREE space you have in your gdrive? doesnt make sense that 10gb is not enough.

**Comment by TheLastBen** [comment]
start the notebok, skip the gdrive cell, try training a v2.1 model for 20 steps, once done training, rerun the session cell and let the model convert, let me know if it converts correctly
